### PR TITLE
About us section

### PR DESCRIPTION
**Pull request by patkarns**
Chrome: chrome

Opera: opera

Firefox: Screen Shot 2020-08-21 at 4 10 19 PM

Safari: 
Screen Shot 2020-08-21 at 4 10 48 PM

Small screens (skytrain hidden)
chrome_small

### CODE REVIEW

File: scss/sections/about-us.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    left: 0;
+    bottom: -45em;
+    overflow: hidden;
+}
+
+@media (max-width: 800px) {
+    #full-skytrain {
+        display: none;
+    }
+}
```
Comment (acchiang) [inline scss/sections/about-us.scss]: @patkarns does this mean we are not showing the skytrain at all if we're in mobile?

File: index.html [hunk 2]
```diff
@@ -91,6 +92,89 @@
...
+    <!-- About Us Section -->
+    <div class="about-us pb-5">
+      <div class="about-us-container container-sm">
+        <h1 class="pb-5">What is CUCSC?</h1>
+        <p>The Canadian Undergraduate Computer Science Conference is an annual, national conference centered around Computer Science research by Canadian undergraduate students and their research teams. </p>
+        <p>Our vision is to promote a trans-disciplinary collaborative environment to broaden the horizons of undergraduate Computer Scientists and all our guests.</p>
+
+    </div>
+  
+    <svg id="full-skytrain" class="mb-2" width="5017" height="2017" viewBox="0 0 5017 2017" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (acchiang) [inline index.html]: Please export the svg file from figma into a png at 2x size, we should use a png file for now if you're not planning to add animations in this iteration! 
Then upload and import the png file into the repo as `` for now. You can find an example in commit `0f3905a70584993f777406c821610707e74fb212`. 

We can all look into transitions and animations later on :)

**Review by acchiang — COMMENTED** [review]
If it's not too much work, please merge master branch into your feature branch as well! Thanks :) 
There are also merge conflicts, let me know if you'd like me to take a look with you

File: scss/sections/about-us.scss (hunk 1 above)
Comment (patkarns) [reply scss/sections/about-us.scss]: I think that's what we wanted for the 2020 landing page. Do we wanna do something different for the full site? We could just shrink the image or show the skytrain without the tracks.

**Comment by acchiang** [comment]
Thanks for making the changes @patkarns! I'm still seeing a placement issue when we're at the smallest desktop screen size, can you adjust this? Maybe add more styles in the same @media tag that hides the skytrain png file when we reach a certain screen size. 
image

**Comment by patkarns** [comment]
@acchiang does this look better?

Screen Shot 2020-08-30 at 4 40 36 PM
Screen Shot 2020-08-30 at 4 40 21 PM

**Review by acchiang — APPROVED** [review]
Looks great @patkarns! Thank you for making all the changes!

**Pull request merged**